source/v4/system.yaml

```diff
@@ -14,3 +14,22 @@
             $ref: "#/definitions/StatusOK"
         '400':
           $ref: '#/responses/BadRequest'
+
+  '/config':
+    get:
+      tags:
+        - system
+      summary: Get the Server configuration
+      description: |
+        Retrieve the current server configuration
```

Comment:
Can the summary be `Get configuration` and add a period to the end of the description